testsuite/testsuite-camel-other/pom.xml

```diff
@@ -56,4 +56,12 @@
     </dependency>
   </dependencies>
 
+  <repositories>
+    <repository>
+      <!-- for restlet -->
+      <id>springsource-libs-releases</id>
+      <url>http://repo.springsource.org/libs-release-remote</url>
+    </repository>
+  </repositories>
```

Comment:
The Camel restlet uses a different repository: https://github.com/apache/camel/blob/master/components/camel-restlet/pom.xml#L40-L46

Can we somehow make it available in central?